### ISSUE TITLE
Apache Storm's storm.py doesn't support Python 3

### ISSUE THREAD

**Issue by macheins** [Parsely/streamparse#267]
`sparse run` is not executable with Python 3 since `print` is a function, not a statement.

```
>>> sparse quickstart wordcount
>>> cd wordcount
>>> sparse run

File "/storm/bin/storm.py", line 94
    print "ERROR:  JAVA_HOME is invalid.  Could not find bin/java at %s." % JAVA_HOME
                                                                        ^
SyntaxError: Missing parentheses in call to 'print'
```

**Comment by Darkless012** [comment]
It is problem of Apache Storm, not streamparse.

```
sudo find / -iname "storm.py"
/opt/apache-storm-1.0.1/bin/storm.py
```

I agree that this should be fixed (it is single print statement without parentheses).

For temporary fix, please change your local file ( enclose the line 94 print statement with parantheses )

**Comment by Darkless012** [comment]
Created pull request https://github.com/apache/storm/pull/1452

**Comment by macheins** [comment]
Sorry for that mistake - I didn't dig deeper yet - and thank you for creating the pull request.

**Comment by Darkless012** [comment]
Its not mistake at all, thank you for creating issue. Result for users are "streamparse is not running". Which should be definitely fixed (even though in different project).

**Comment by HeartSaVioR** [comment]
Sorry for the long delay. next bugfix release (1.0.2) will include patch on @Darkless012 .

**Comment by HeartSaVioR** [comment]
@macheins @Darkless012 
Could you check it's resolved with Storm 1.0.2?

**Comment by macheins** [comment]
Seems to work now. Thank you @HeartSaVioR
